### DISCUSSION TITLE
setFeedback event not working.

### DISCUSSION THREAD

**Discussion by nickhudson4** [elgatosf/elgato#58] [Help]
I believe this is a bug since I have seen another report of setFeedback not working past version 6.1.

But I am basically trying to change dial icon and title using the setFeedback event. I have tried using the built in dial layout as well as a custom one. I am able to verify that other events work fine (setImage, setTitle, etc.). setFeedback simply does nothing.

Curious if anyone else has experienced this or has a solution?

**Comment by GeekyEggo** [comment]
Hey @nickhudson4,

To help diagnose what's happening we'll need some more information; specifically, could you please provide:

- The exact Stream Deck version and operating system you're using.
- The name of the library you're using to communicate with Stream Deck (if any).
- The manifest of your plugin, specifically the `Action` you're trying to set the feedback for.
- The JSON that represents the feedback layout.
- Information into how you're trying to update the feedback.

**Reply by GeekyEggo** [reply]
Hi @nickhudson4 just following up on this to see if you're still experiencing issues with the `setFeedback` event before I close this discussion?
